sdk/tests/tutorials/ibor/test_valuation.py

```diff
@@ -59,40 +58,61 @@ def test_portfolio_aggregation(self):
-
         prices = [
             models.InstrumentAnalytic(self.instrument_ids[0], 100),
             models.InstrumentAnalytic(self.instrument_ids[1], 200),
             models.InstrumentAnalytic(self.instrument_ids[2], 300)
         ]
 
-        #   add prices
-        self.analytic_stores_api.set_analytics(scope=TestDataUtilities.tutorials_scope,
-                                               year=effective_date.year,
-                                               month=effective_date.month,
-                                               day=effective_date.day,
-                                               data=prices)
+        requests = []
+        for ii in range(3):
+            requests.append(
+                models.UpsertQuoteRequest(
+                    quote_id=models.QuoteId(
+                        models.QuoteSeriesId(
+                            provider="DataScope",
+                            instrument_id=self.instrument_ids[ii],
+                            instrument_id_type="LusidInstrumentId",
+                            quote_type="Price",
+                            field="mid"
+                        ),
+                        effective_at=datetime(2019, 4, 15, tzinfo=pytz.utc)
+                    ),
+                    metric_value=models.MetricValue(
+                        value=prices[ii].value,
+                        unit="GBP"
+                    )
+                )
+            )
+
+        # for request_number in range(len(requests)):
+        self.quotes_api.upsert_quotes(TestDataUtilities.tutorials_scope,
+                                      quotes={"quote" + str(request_number): requests[request_number]
+                                              for request_number in range(len(requests))})
+
+        inline_recipe = models.ConfigurationRecipe(
+            code='quotes_recipe',
+            market=models.MarketContext(
+                market_rules=[],
+                suppliers=models.MarketContextSuppliers(
+                    equity='DataScope'
+                ),
+                options=models.MarketOptions(
+                    default_supplier='DataScope',
+                    default_instrument_code_type='LusidInstrumentId',
+                    default_scope=TestDataUtilities.tutorials_scope)
+            )
+        )
 
         aggregation_request = models.AggregationRequest(
-            recipe_id=models.ResourceId(TestDataUtilities.tutorials_scope, "default"),
+            inline_recipe=inline_recipe,
             metrics=[
                 models.AggregateSpec("Instrument/default/Name", "Value"),
                 models.AggregateSpec("Holding/default/PV", "Proportion"),
                 models.AggregateSpec("Holding/default/PV", "Sum")
             ],
             group_by=["Instrument/default/Name"],
-            effective_at=effective_date
+            effective_at=datetime(2019, 4, 19, tzinfo=pytz.utc)  # effective_date
```

Comment:
Should use the `effective_date` variable

Reply:
fixed